IntegrationTests/allocation-counter-tests-framework/run-allocation-counter.sh

```diff
@@ -61,8 +61,8 @@ EOF
     cat <<EOF
     ],
     dependencies: [
-        .package(url: "HookedFunctions/", branch: "main"),
-        .package(url: "AtomicCounter/", branch: "main"),
+        .package(path: "HookedFunctions"),
+        .package(path: "AtomicCounter"),
         .package(url: "$swiftpm_pkg_name/", branch: "main"),
```

Comment:
Is this one local too?

Reply:
Yeah, changing it took a little more effort than expected though.